### PR TITLE
Fix typos in the tBTC docs

### PR DESCRIPTION
**Pull request by michalinacienciala**
Fixing a couple of typos spotted during reading of the tBTC documentation.

### CODE REVIEW

File: docs/bonding/index.adoc [hunk 1]
```diff
@@ -190,11 +190,11 @@ supply is in line with BTC custodied by TBTC-backing deposits.]
 Forced liquidation should be rare, as rational signers will redeem deposits
 before liquidation becomes necessary. However, the possibility of extreme
 punishment via liquidation is necessary to prevent dishonest behavior from
-signers. Liquidation may occur because because signers
```
Comment (Shadowfiend) [inline docs/bonding/index.adoc:193]: Oh man a double-word, classic error!

File: docs/deposits/index.adoc [hunk 2]
```diff
@@ -156,7 +156,7 @@ The distributed key generation protocol should result in three properties:
 2. Each member of the signing group should have a _threshold ECDSA secret key
    share_{threshold-signature}, which can be used to create a
    _threshold ECDSA signature share_ for any transactions involving the
-   signing group's wallet.
+   signing of the group's wallet.
```
Comment (Shadowfiend) [inline docs/deposits/index.adoc:159]: This was correct in the original form. These are transactions that involve the wallet that belongs to the signing group (i.e., involving the signing group's wallet).

**Review by Shadowfiend — DISMISSED** [review]
One small one that needs to be reverted, rest looks good!

File: docs/deposits/index.adoc (hunk 2 above)
Comment (michalinacienciala) [reply docs/deposits/index.adoc]: You're absolutely right, when I look at it today I don't know how I managed not to understand it in this sentence (when I got it in the other places)...
Anyway, fixed in cc51ca1e314fcb8224242e466e46e27beacd64f5.

**Review by mhluongo — APPROVED** [review]
LGTM!

**Pull request merged**